ts/src/pro/binance.ts

```diff
@@ -3384,7 +3384,7 @@ export default class binance extends binanceRest {
         const clientOrderId = this.safeStringN (params, [ 'clientAlgoId', 'origClientOrderId', 'clientOrderId' ]);
         if (clientOrderId !== undefined) {
             payload['origClientOrderId'] = clientOrderId;
-        } if (market['linear'] && market['swap'] && isConditional) {
+        } else if (market['linear'] && market['swap'] && isConditional) {
```

Comment:
I assume there should have been `else if` (as opposed to a new `if` clause starting from a new line)